coralproject/talk
# Username Permissions

## What does this PR do?

- Blocks users from interacting if they are in the username change flow
- Changes out strings for string constants imported to simplify the build/import process

## How do I test this PR?

1. Have your username rejected and changed (but not approved)
2. Try and click "Respect", notice no error, but no interaction either
